api-server/server/boot/settings.js

```diff
@@ -188,18 +188,19 @@ function updateMyAbout(req, res, next) {
 
 function createUpdateMyUsername(app) {
   const { User } = app.models;
+  console.log(User);
```

Comment:
```suggestion
  
```
Just a reminder comment to remove this later, when you are ready with the PR.

Reply:
@raisedadead removed console.log